apps/dashboard/components/app-users/integrations/user-selection-list.tsx

```
@@ -0,0 +1,134 @@
+    setSelectedUsers(newSelection);
+  };
+
+  // Footer data calculation - adapted for imported users
+  const getFooterValue = (key: string) => {
+    switch (key) {
+      case 'identity':
+        return `${users.length} users`;
+      case 'email':
+        const emailCount = users.reduce((sum, user) => {
+          return sum + (user.email ? 1 : 0);
+        }, 0);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)